symengine/compatibility.py

```diff
@@ -7,6 +7,11 @@
 
 import operator
 from collections import defaultdict
+import sys
+import os
+from collections import namedtuple
+from functools import update_wrapper
+from threading import RLock
```

Comment:
Should the entire file remain untouched or should the import related changes be?